build/ruby/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ruby
```

Comment:
what do you think about pinning? Shall we try to do it now or not?

Reply:
Sure, let's try it now. Added one here. Can do the others in another PR.